### ISSUE TITLE
Heroku: 'Is webpack still compiling?'

### ISSUE THREAD

**Issue by dino-k-29** [rails/webpacker#279]
Getting the following error `ActionView::Template::Error (Can't find welcome/index.js in /app/public/packs/manifest.json. Is webpack still compiling?):` on Heroku with this project: https://github.com/dino-k-29/rails-vue-webpacker

I'm using the following buildpacks:
```
1. heroku/nodejs
2. heroku/ruby
```

**Comment by gauravtiwari** [comment]
@dino-k-29 Does that reference exist in `manifest.json` when you run `bundle exec rails webpacker:compile` locally?

**Comment by dino-k-29** [comment]
@gauravtiwari yes, like so:

```
{
  "welcome/index.js": "http://localhost:8080/packs/welcome/index.js",
  "welcome/index.js.map": "http://localhost:8080/packs/welcome/index.js.map"
}
``

**Comment by gauravtiwari** [comment]
@dino-k-29 Ah, so you have compiled assets locally and therefore, heroku assets:precompile hook doesn't get run. 

```
-----> Preparing Rails asset pipeline
       Detected manifest.yml, assuming assets were compiled locally
```

More info here - https://devcenter.heroku.com/articles/rails-asset-pipeline#no-debug-output-at-all

Delete the assets folder and try again. If you rather compile it locally, please remove packs from `.gitignore` so both can be committed to source control.

**Comment by gauravtiwari** [comment]
And the second problem is you are using es6 syntax, related to - https://github.com/rails/webpacker/issues/280

**Comment by gauravtiwari** [comment]
es6 syntax issue should be fixed by #291

**Comment by gauravtiwari** [comment]
Closing this one as the first one isn't an issue and the es6 problem is been tracked by #280
